src/clts2vec/parse.py

```diff
@@ -103,23 +109,28 @@ def get_vec(self, sound, vectorize=True):
             >>> print(vec["cont"]
             - 1
         """
-        
-        sound = self.validate(sound)        
-        base_vec = Vector(self.binary_features) #{f: 0 for f in self.binary_features}
+
+        sound = self.validate(sound)
+        base_vec = Vector(self.binary_features)  # {f: 0 for f in self.binary_features}
 
         # check if sound is diphthong or complex
         complex_sound = False
+        sound_to_sound = ""
         if sound.endswith(" diphthong"):
+            self._apply_feature("diphthong", base_vec)
             sound_to_sound = sound[sound.index(" to ") + 4: -10] + " vowel"
             sound = sound[5: sound.index(" to ")] + " vowel"
+            # set up feature pairs for diphthong trajectory
```

Comment:
Ah, I see that you had them explicitly covered in your feature handling.

Reply:
But maybe we don't need that anymore now, and we could just say: dipthong is first vowel and second vowel? So any `from_` feature would not be needed any more?

Reply:
I would prefer to leave it like that, otherwise I would essentially need to change the entire infrastructure for assigning diphthong features...